_posts/en/newsletters/2023-11-15-newsletter.md

```diff
@@ -0,0 +1,121 @@
+---
+title: 'Bitcoin Optech Newsletter #277'
+permalink: /en/newsletters/2023/11/15/
+name: 2023-11-15-newsletter
+slug: 2023-11-15-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes an update to the proposal for ephemeral
+anchors and provides a contributed field report about miniscript from a
```

Comment:
I've always seen "Miniscript" being capitalized, as pretty much all programming languages are:
```suggestion
anchors and provides a contributed field report about Miniscript from a
```
This would obviously need to be changed throughout the contrib post.

Reply:
We tend to lowercase miniscript in the newsletter. 

Although its fair to say maybe I trampled on Antoines writeup with our Optech style preferences :)